lang/en.ts

```diff
@@ -3453,4 +3425,60 @@ export const lib = {
   oauth: {
     missingClientId: 'Error building oauth URL: missing client ID.',
   },
+  migrate: {
+    componentsToBeMigrated: components =>
```

Comment:
I'm assuming that this is just temporary copy, until a designer can take a look, correct?

Reply:
This is the UX reviewed copy, I just relocated it to this file.